README.md

```diff
@@ -147,6 +148,8 @@ Compiling YCM **without** semantic support for C-family languages:
 
 If you want semantic C# support, you should add `--omnisharp-completer` to the
 install script as well. If you want Go support, you should add `--gocode-completer`.
+If you want semantic TypeScript support, install the TypeScript SDK with npm
+`npm install -g typescript`
```

Comment:
You should probably also add this line to the Ubuntu section, which mirrors the Mac section.